### PR TITLE
add docker image flag to pre-commit

### PR DESCRIPTION
**Pull request by ilaner**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10338

### CODE REVIEW

File: demisto_sdk/commands/pre_commit/hooks/docker.py [hunk 1]
```diff
@@ -101,13 +91,29 @@ def docker_tag_to_runfiles(
     Returns: A dict of image to List of files(Tuple[path, obj]) including native images
 
     """
+    docker_flags = set(docker_image_flag.split(","))
     tags_to_files = defaultdict(list)
     for file, obj in files_to_run:
         if not obj:
```
Comment (dorschw) [inline demisto_sdk/commands/pre_commit/hooks/docker.py:97]: would be more readable as `if obj: for image in ...` `else: continue`

File: demisto_sdk/commands/pre_commit/hooks/docker.py [hunk 2]
```diff
@@ -101,13 +91,29 @@ def docker_tag_to_runfiles(
...
+
+
+@lru_cache()
+def get_docker_python_path() -> str:
+    """
+    precommit by default mounts the content repo to source.
+    This means CommonServerPython's path is /src/Packs/Base/...CSP.py
+    Returns: A PYTHONPATH formatted string
+    """
+    path_to_replace = str(Path(CONTENT_PATH).absolute())
```
Comment (dorschw) [inline demisto_sdk/commands/pre_commit/hooks/docker.py:112]: absolute compared to cwd?

Comment (ilaner) [reply demisto_sdk/commands/pre_commit/hooks/docker.py]: this is not related to this PR, moved to its original place to prevent the git diff

**Review by JudahSchwartz — APPROVED** [review]
amazing

**Pull request merged**